<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget_pages">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1200</width>
      <height>800</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>8</number>
    </property>
    <widget class="QWidget" name="page_home">
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>1201</width>
        <height>81</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_home_menu">
       <property name="leftMargin">
        <number>200</number>
       </property>
       <property name="rightMargin">
        <number>200</number>
       </property>
       <property name="horizontalSpacing">
        <number>40</number>
       </property>
       <item row="0" column="2">
        <widget class="QPushButton" name="pushButton_home_food">
         <property name="text">
          <string>Food Options</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="pushButton_home_cities">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Available Cities</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="pushButton_home_login">
         <property name="text">
          <string>Admin Login</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton_home_exit">
         <property name="text">
          <string>Exit</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>300</y>
        <width>1201</width>
        <height>231</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_home_trips">
       <property name="leftMargin">
        <number>500</number>
       </property>
       <property name="rightMargin">
        <number>500</number>
       </property>
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton_home_berlin">
         <property name="text">
          <string>Berlin Trip</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButton_home_paris">
         <property name="text">
          <string>Paris Trip</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="pushButton_home_custom">
         <property name="text">
          <string>Custom Trip</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_home_title">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>HOME PAGE</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_cities">
     <widget class="QTableView" name="tableView_cities_view">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>100</y>
        <width>600</width>
        <height>600</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_cities_back">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>80</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_cities_title">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>CITIES PAGE</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_food">
     <widget class="QTableView" name="tableView_food_view">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>100</y>
        <width>600</width>
        <height>600</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_food_back">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>80</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_food_title">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>FOOD PAGE</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_paris">
     <widget class="QSpinBox" name="spinBox_paris_select">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>220</y>
        <width>71</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_paris_select">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>200</y>
        <width>281</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Select the number of cities you want to travel to</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_paris_title">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>PARIS PAGE</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView_paris_cities">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>100</y>
        <width>600</width>
        <height>600</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_paris_back">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>80</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_paris_continue">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>300</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Continue</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_custom">
     <widget class="QPushButton" name="pushButton_custom_back">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>80</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_custom_title">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>CUSTOM PAGE</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_custom_continue">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>300</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Continue</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_custom_selectstart">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>200</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Select your starting city</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_custom_select">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>220</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget_custom_select">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>100</y>
        <width>600</width>
        <height>600</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_custom_selectcities">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>80</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Select cities to visit</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_purchase">
     <widget class="QLabel" name="label_purchase_title">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>PURCHASE PAGE</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_purchase_back">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>80</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_purchase_continue">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>650</y>
        <width>120</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Confirm Purchase</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget_purchase_pos">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>120</y>
        <width>1040</width>
        <height>500</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_purchase_total">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>650</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Total: </string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_receipt">
     <widget class="QLabel" name="label_receipt_title">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>RECEIPT PAGE</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_receipt_back">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>650</y>
        <width>120</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Back to Home</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView_receipt_view">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>120</y>
        <width>1040</width>
        <height>500</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_receipt_distance">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>650</y>
        <width>200</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Total Distance: </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_receipt_spent">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>680</y>
        <width>200</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Total Spent:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_login">
     <widget class="QPushButton" name="pushButton_login_continue">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>410</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_login_username">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>300</y>
        <width>60</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Username:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_login_password">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>350</y>
        <width>60</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Password:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_login_back">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>80</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Home</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_username_password">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>350</y>
        <width>200</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_login_username">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>300</y>
        <width>200</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_login_error">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>260</y>
        <width>191</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>username/password incorrect :)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_login_title">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>LOGIN PAGE</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_admin">
     <widget class="QLabel" name="label_admin_title">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>ADMIN PAGE</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_admin_back">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>80</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Log Out</string>
      </property>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>270</y>
        <width>291</width>
        <height>86</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="0">
        <widget class="QLabel" name="label_admin_city">
         <property name="text">
          <string>City: </string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_admin_food">
         <property name="text">
          <string>Food Name: </string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_admin_price">
         <property name="text">
          <string>Food Price: </string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit_admin_food"/>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEdit_admin_price"/>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEdit_admin_city"/>
       </item>
      </layout>
     </widget>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>100</y>
        <width>600</width>
        <height>600</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>170</y>
        <width>254</width>
        <height>80</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton_admin_add">
         <property name="text">
          <string>Add Food</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="pushButton_admin_edit">
         <property name="text">
          <string>Edit Price</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="pushButton_admin_delete">
         <property name="text">
          <string>Delete Food</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>130</y>
        <width>171</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Import Cities From File</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
